src/hx711-element.ts

```diff
@@ -0,0 +1,402 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ElementPin, GND, VCC } from './pin';
+
+@customElement('wokwi-hx711')
+export class HX711Element extends LitElement {
+  @property() type: '5kg' | '50kg' | 'gauge' | undefined;
+  @property() width = '58mm';
+  @property() height = '43mm';
```

Comment:
width/height shouldn't be provided by the user. Instead, they should be derived from the type. 

One way to do this would be to introduce a getter, e.g.:

```ts
get dimensions() {
  switch (this.type) { 
    case '5kg': return {width: '58mm', height:'43mm'};
    ...
  }
}
```

Then, in render, you could do
```ts
 const { width, height } = this.dimensions
```